internal/kuberesource/sets.go

```diff
@@ -470,15 +445,12 @@ func Emojivoto(smMode serviceMeshMode) []any {
 		return resources
 	}
 
-	if smProxyEmoji != nil {
-		AddServiceMesh(emoji, smProxyEmoji)
-	}
-	if smProxyWeb != nil {
-		AddServiceMesh(web, smProxyWeb)
-	}
-	if smProxyVoting != nil {
-		AddServiceMesh(voting, smProxyVoting)
-	}
+	emoji.WithAnnotations(map[string]string{smIngressConfigAnnotationKey: ""})
+	voting.WithAnnotations(map[string]string{smIngressConfigAnnotationKey: ""})
+	web.WithAnnotations(map[string]string{
+		smIngressConfigAnnotationKey: smWebIngress,
+		smEgressConfigAnnotationKey:  smWebEgress,
+	})
```

Comment:
This should be skipped in case `ServiceMeshDisabled`, shouldn't it?

Reply:
In this case, the resources are returned above without the change.